static/app/views/replays/utils/useReplayEvent.tsx

```diff
@@ -22,7 +23,7 @@ type State = {
   /**
    * The root replay event
    */
-  event: undefined | Event;
+  event: undefined | EventTransaction;
```

Comment:
Make the types more specific. Event is a combination of `EventError | EventTransaction | EventBase` so we can use the specific `EventTransaction` here and benefit from it having `startTimestamp` and `endTimestamp` as required fields.